controllers/profile.js

```diff
@@ -0,0 +1,95 @@
+import _ from 'underscore';
+import { User } from '../models';
+import splitName from '../helpers/splitName';
+
+/**
+ * @class ProfileController
+ *  @override
+ * @export
+ *
+ */
+export default class ProfileController {
+  /**
+   * @description - Renders a user's profile
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberof ProfileController
+   *
+   * @returns {object} User Profile
+   */
+  static async viewProfile(req, res) {
+    const { id } = req.user;
+
+    try {
+      const foundUser = await User.findOne({
+        raw: true,
+        where: {
+          id
+        }
+      });
+      delete foundUser.password;
+      delete foundUser.verificationId;
+
+      const splitNamesObject = splitName(foundUser);
+
+      return res.status(200).json({
+        success: true,
+        body: _.extendOwn(foundUser, splitNamesObject)
```

Comment:
@leksyib what does this do?

Reply:
It returns a new object which is a combination of the `foundUser` and `splitNamesObject` objects. https://underscorejs.org/#extendOwn